pkg/stores/partition/store.go

```diff
@@ -384,7 +385,7 @@ func toAPIEvent(apiOp *types.APIRequest, schema *types.APISchema, event watch.Ev
 		return apiEvent
 	}
 
-	apiEvent.Object = toAPI(schema, event.Object)
+	apiEvent.Object = toAPI(schema, event.Object, nil)
```

Comment:
In the first version of this, the `toAPIEvent` function accepted warnings, are they not needed any more for events?

Reply:
I don't think it's needed/useful in our case.  I'll keep exploring though.  It might be back.